meilisearch-http/src/error.rs

```diff
@@ -74,7 +74,7 @@ impl IntoResponse for ResponseError {
             ResponseError::Internal(err) => {
                 error!("internal server error: {}", err);
                 error(
-                    String::from("Internal server error"),
+                    format!("Internal server error:{:?}", err),
```

Comment:
That was done on purpose, we don't want to expose the internal server error to the user.
The error message can be seen in the logs if needed.